pkg/rpc/response/result/invoke.go

```diff
@@ -171,41 +201,14 @@ func (r *Invoke) UnmarshalJSON(data []byte) error {
 				break
 			}
 			if st[i].Type() == stackitem.InteropT {
-				iteratorAux := new(iteratorAux)
-				if json.Unmarshal(arr[i], iteratorAux) == nil {
-					if len(iteratorAux.Interface) != 0 {
-						if iteratorAux.Interface != iteratorInterfaceName {
-							err = fmt.Errorf("unknown InteropInterface: %s", iteratorAux.Interface)
-							break
-						}
-						var iID uuid.UUID
-						iID, err = uuid.Parse(iteratorAux.ID) // iteratorAux.ID is always non-empty, see https://github.com/neo-project/neo-modules/pull/715#discussion_r897635424.
-						if err != nil {
-							err = fmt.Errorf("failed to unmarshal iterator ID: %w", err)
-							break
-						}
-						// It's impossible to restore initial iterator type; also iterator is almost
-						// useless outside the VM, thus let's replace it with a special structure.
-						st[i] = stackitem.NewInterop(Iterator{
-							ID: &iID,
-						})
-					} else {
-						iteratorValues := make([]stackitem.Item, len(iteratorAux.Value))
-						for j := range iteratorValues {
-							iteratorValues[j], err = stackitem.FromJSONWithTypes(iteratorAux.Value[j])
-							if err != nil {
-								err = fmt.Errorf("failed to unmarshal iterator values: %w", err)
-								break
-							}
-						}
-						// It's impossible to restore initial iterator type; also iterator is almost
-						// useless outside the VM, thus let's replace it with a special structure.
-						st[i] = stackitem.NewInterop(Iterator{
-							Values:    iteratorValues,
-							Truncated: iteratorAux.Truncated,
-						})
-					}
+				var iter = Iterator{}
+				err = json.Unmarshal(arr[i], &iter)
+				if err != nil {
+					break
 				}
+				// It's impossible to restore initial iterator type; also iterator is almost
+				// useless outside the VM, thus let's replace it with a special structure.
```

Comment:
If we're changing it, maybe adjust the comment along the way? Iterators are not useless outside the VM anymore.

Reply:
OK.